sphinxcontrib/confluencebuilder/translator/storage.py

```diff
@@ -768,7 +784,17 @@ def visit_tabular_col_spec(self, node):
         raise nodes.SkipNode
 
     def visit_colspec(self, node):
-        raise nodes.SkipNode
+        # visit colspec node to identify stub-column,
+        # but don't produce a <colspec> tag
+        # see https://svn.code.sf.net/p/docutils/code/trunk/docutils/docutils/writers/_html_base.py
+        self.colspecs.append(node)
+        node.parent.stubs.append(node.attributes.get('stub'))
+
+    def depart_colspec(self, node):
+        # see https://svn.code.sf.net/p/docutils/code/trunk/docutils/docutils/writers/_html_base.py
+        # because we're not actually implementing colspec,
+        # don't write </colspec> tag
+        pass
```

Comment:
Recommend a series of these comments to be removed in the change since (opinion) the implementation is basic enough that each of these sections do not need individual comments associating them to a feature. This applies to these lines and other related lines.

Something like the following would keep it simpler:

```
    def visit_colspec(self, node):
        self.colspecs.append(node)
        node.parent.stubs.append(node.attributes.get('stub'))

    def depart_colspec(self, node):
        pass
```

And in the commit message, reference the original implementation reference:

> https://svn.code.sf.net/p/docutils/code/trunk/docutils/docutils/writers/_html_base.py